pkg/core/native/policy.go

```diff
@@ -127,6 +132,11 @@ func newPolicy() *Policy {
 	md = newMethodAndPrice(p.unblockAccount, 1<<15, callflag.States)
 	p.AddMethod(md, desc)
 
+	desc = newDescriptor("setSystemFeeRefundCost", smartcontract.VoidType,
```

Comment:
Could you, please, update the native Policy interop wrapper (https://github.com/nspcc-dev/neo-go/blob/master/pkg/interop/native/policy/policy.go)?